Patch to support US-country code missing in SIP register form

bdeanindy opened the pull request:
This works for now to make sure either with or without the US-country code of '1' that the SIP register will still happen.

grokify commented:
Can we remove non 0-9 characters so we can support human readable phone numbers at the same time:

* 650-555-1212
* (650) 555-1212

bdeanindy commented:
@grokify I have updated  the commit to include the change you recommended as well as I saw that I was not checking for the plus symbol in my inclusion. Does this work better and does it resolve #44 ?

grokify commented:
Is it possible to write a function like `format_number_e164_us()`. This would better show users what are are attempting to accomplish for them to follow our code. This way they can understand what we are doing to replace it with their own version.

Here's a quick example I put together: https://jsfiddle.net/me1xsdqk/1/

kirill-konshin commented:
Is PR ready to be merged?

grokify commented:
Not yet, should at least have the following change:

```javascript
// current
username.replace(/\W/g, '');
// change
username = username.replace(/\W/g, '');
```

bdeanindy commented:
Fixing this now and will push in a moment.

bdeanindy commented:
Sorry for the funk in there...old habits die hard, and I did a merge upstream/master instead of a rebase.

I reverted the merge commit and squashed.

Let me know if that works, if not, I'll fix per recommendation.
:)

kirill-konshin commented:
Right now I see 5 commits in this PR. Plus I've just merged @vyshakhbabji other PR. So please rebase and squash.

bdeanindy commented:
ACK! OK.

kirill-konshin commented:
I am closing this PR because the next version will be based on `clean` branch. Please make a new PR.
